src/Components/CardListStyles.css

```diff
@@ -1,17 +1,31 @@
-.list-container{
```

Comment:
styles on Style folder

Reply:
Yes, I thought about it, I don't know what they're doing there. But since it was not me who made those files, I was afraid of breaking the code of the partner who made it when moving them. I'll try moving them and see if anything breaks.

Reply:
Done! Of course, several files were added to the PR since several imported the css folders and I had to change the routes to all of them, otherwise everything would be broken.